### PR TITLE
Docs: Improve wording of the FAQ

### PR DESCRIPTION
**Pull request by hariom1625**
A slight update in faq.md for better conveying of the information to make it more understandable.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -9,16 +9,16 @@ need to make a change to your build steps, you are forced to make that change ac
 your similar projects. Using a preset rather than a boilerplate keeps this process DRY.
 
 Tools like [Create React App](https://github.com/facebook/create-react-app) have
-been fantastic improvements to the tooling ecosystem, but unfortunately only work on specific
-environments like React, and do not allow simple extensibility of the build configuration. To
+been fantastic improvements to the tooling ecosystem, but unfortunately, only work on specific
```
Comment (edmorley) [inline docs/faq.md:12]: I think this sentence is more correct as it was before?
```suggestion
been fantastic improvements to the tooling ecosystem, but unfortunately only work on specific
```

File: docs/faq.md [hunk 2]
```diff
@@ -9,16 +9,16 @@ need to make a change to your build steps, you are forced to make that change ac
...
+environments like React, and do not allow simple extensibility of the build configuration. To
```
Comment (edmorley) [inline docs/faq.md:13]: Trailing whitespace was added here?

```suggestion
environments like React, and do not allow simple extensibility of the build configuration. To
```

File: docs/faq.md [hunk 3]
```diff
@@ -9,16 +9,16 @@ need to make a change to your build steps, you are forced to make that change ac
 your similar projects. Using a preset rather than a boilerplate keeps this process DRY.
 
 Tools like [Create React App](https://github.com/facebook/create-react-app) have
-been fantastic improvements to the tooling ecosystem, but unfortunately only work on specific
-environments like React, and do not allow simple extensibility of the build configuration. To
+been fantastic improvements to the tooling ecosystem, but unfortunately, only work on specific
+environments like React, and do not allow simple extensibility of the build configuration. To 
 answer this, new and similar projects are cropping up to build different types of projects,
-often duplicating efforts which miss out on the best practices to share with the other project
+often duplicating efforts that miss out on the best practices to share with the other project
```
Comment (edmorley) [inline docs/faq.md:15]:
```suggestion
often duplicating efforts and missing out on opportunities to share best practices with the other project
```

File: docs/faq.md [hunk 4]
```diff
@@ -38,9 +38,9 @@ For example, let's say that a preset instead used the webpack object configurati
 config.plugins.push(new webpack.EnvironmentPlugin(['NODE_ENV']));
 ```
 
-If you wanted to extend this plugin in your own project to add more environment variables, you would most likely
-resort to either adding a new instance of the `EnvironmentPlugin` by requiring webpack yourself, or looping through
-webpack's plugin array, removing the plugin, and re-instantiating it with your own arguments.
+If you wanted to extend this plugin in your project to add more environment variables, you would most likely
+resort to either adding a new instance of the EnvironmentPlugin by requiring webpack yourself, or looping through
+webpack's plugin array, removing the plugin, and re-instantiating it with your arguments.
```
Comment (edmorley) [inline docs/faq.md:43]: These two changes regress the readability IMO. Reverting back:
```suggestion
resort to either adding a new instance of the `EnvironmentPlugin` by requiring webpack yourself, or looping through
webpack's plugin array, removing the plugin, and re-instantiating it with your own arguments.
```

File: docs/faq.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ config.plugins = config.plugins.map(plugin => {
   }
   
   return new webpack.EnvironmentPlugin([...plugin.keys, ...customEnvironmentVariables]);
-});
+});
```
Comment (edmorley) [inline docs/faq.md:52]: Trailing whitespace was added.

```suggestion
});
```

File: docs/faq.md [hunk 6]
```diff
@@ -61,18 +61,18 @@ possible when working with raw webpack configuration data.
 
 ### Can I just re-use my existing webpack configuration?
 
-The Neutrino configuration does let you merge a configuration object, but does not accept a normal webpack configuration
+The Neutrino configuration does let you merge a configuration object but does not accept a normal webpack configuration
```
Comment (edmorley) [inline docs/faq.md:64]: I prefer this the way it was before :-)

```suggestion
The Neutrino configuration does let you merge a configuration object, but does not accept a normal webpack configuration
```

File: docs/faq.md [hunk 7]
```diff
@@ -61,18 +61,18 @@ possible when working with raw webpack configuration data.
...
 out of the box. Since Neutrino uses webpack-chain and enforces naming of plugins, rules, and loaders, you must use an
 object that corresponds with this "schema". Typically this would involve transforming your webpack configuration to nest
 entities requiring a name into an object which maps the name to the entity.
 
-In short, you must transform your webpack configuration in order to merge it into the Neutrino configuration cleanly.
+In short, you must transform your webpack configuration to merge it into the Neutrino configuration cleanly.
```
Comment (edmorley) [inline docs/faq.md:69]: This was correct usage before IMO

```suggestion
In short, you must transform your webpack configuration in order to merge it into the Neutrino configuration cleanly.
```

File: docs/faq.md [hunk 8]
```diff
@@ -61,18 +61,18 @@ possible when working with raw webpack configuration data.
...
 
 ### What is the difference between middleware and presets?
 
-The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality is in
+The term "presets" are a hold-over from Neutrino pre-v5. Presets are middleware, the difference, in reality, is in
```
Comment (edmorley) [inline docs/faq.md:73]: The bolding of "are" was intentional here and the previous wording I think was slightly better. Reverting:

```suggestion
The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality is in
```

File: docs/faq.md (hunk 2 above)
Comment (hariom1625) [reply docs/faq.md:13]: While editing it might have got added I didn't observe 😅

File: docs/faq.md (hunk 4 above)
Comment (hariom1625) [reply docs/faq.md:43]: In this, the use of 'own' isn't necessary as its been stated before.

**Comment by hariom1625** [comment]
Thanks for reviewing the PR and suggesting the improvements I will commence some changes and make a pull request.

File: docs/faq.md [hunk 9]
```diff
@@ -38,9 +38,9 @@ For example, let's say that a preset instead used the webpack object configurati
 config.plugins.push(new webpack.EnvironmentPlugin(['NODE_ENV']));
 ```
 
-If you wanted to extend this plugin in your own project to add more environment variables, you would most likely
+If you wanted to extend this plugin in your project to add more environment variables, you would most likely
 resort to either adding a new instance of the `EnvironmentPlugin` by requiring webpack yourself, or looping through
-webpack's plugin array, removing the plugin, and re-instantiating it with your own arguments.
+webpack's plugin array, removing the plugin, and re-instantiating it with your arguments.
```
Comment (edmorley) [inline docs/faq.md:43]: I think the existing wording was better here, as mentioned before:
https://github.com/neutrinojs/neutrino/pull/1592#discussion_r456028828

"own" is referring to two different things, so isn't duplicating, and it's important to the meaning of the sentence. The key part of this paragraph is that how you use code in your "own" project, has different needs from a "shared" project.

Please don't close review comments as "resolved" just because you disagree. That doesn't mean you can't start a discussion to convey your points and try and persuade the reviewer - but shutting down the conversation is a quick way to alienate the project maintainers :-)

File: docs/faq.md [hunk 10]
```diff
@@ -66,13 +66,13 @@ out of the box. Since Neutrino uses webpack-chain and enforces naming of plugins
 object that corresponds with this "schema". Typically this would involve transforming your webpack configuration to nest
 entities requiring a name into an object which maps the name to the entity.
 
-In short, you must transform your webpack configuration in order to merge it into the Neutrino configuration cleanly.
+In short, you must transform your webpack configuration to facilitate its merging into the Neutrino configuration cleanly.
```
Comment (edmorley) [inline docs/faq.md:69]: I think the existing wording was more accurate here. I mentioned this in the previous review comment, which was marked as resolve with no discussion but the change not made?
https://github.com/neutrinojs/neutrino/pull/1592#discussion_r456029776

File: docs/faq.md [hunk 11]
```diff
@@ -66,13 +66,13 @@ out of the box. Since Neutrino uses webpack-chain and enforces naming of plugins
...
 
 ### What is the difference between middleware and presets?
 
-The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality is in
+The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference, in reality, is in
```
Comment (edmorley) [inline docs/faq.md:73]:
```suggestion
The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality, is in
```

**Review by edmorley — CHANGES_REQUESTED** [review]

File: docs/faq.md (hunk 9 above)
Comment (hariom1625) [reply docs/faq.md]: I will review the changes once again and will make the necessary changes. And about the part of me marking the comments as 'resolved' , I am sorry for that as I am quite new and is trying to learn the environment of GitHub, thank you for correcting, I will keep that in mind for future instances.

File: docs/faq.md (hunk 10 above)
Comment (hariom1625) [reply docs/faq.md]: Marking review as resolved was quite a mistake, I should have kept open for discussion. I would commit the changes, as now I think prior was better.

File: docs/faq.md [hunk 12]
```diff
@@ -9,16 +9,16 @@ need to make a change to your build steps, you are forced to make that change ac
 your similar projects. Using a preset rather than a boilerplate keeps this process DRY.
 
 Tools like [Create React App](https://github.com/facebook/create-react-app) have
-been fantastic improvements to the tooling ecosystem, but unfortunately only work on specific
+been fantastic improvements to the tooling ecosystem, but unfortunately, it only works on specific
```
Comment (edmorley) [inline docs/faq.md:12]: Unfortunately "it only works" isn't correct grammar, since the beginning of the sentence starts with "Tools like" which is plural.

This should stay as "only work".

```suggestion
been fantastic improvements to the tooling ecosystem, but unfortunately only work on specific
```

File: docs/faq.md [hunk 13]
```diff
@@ -70,9 +70,9 @@ In short, you must transform your webpack configuration in order to merge it int
 
 ### What is the difference between middleware and presets?
 
-The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality is in
+The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality, is in
```
Comment (edmorley) [inline docs/faq.md:73]:
```suggestion
The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality is in
```

**Review by edmorley — CHANGES_REQUESTED** [review]

**Review by edmorley — APPROVED** [review]
